### PR TITLE
Wait for plugins to load if the stored panel name is not builtin

### LINKED ISSUE #965: Restore sidebar panel selection earlier
If the stored selected sidebar panel is "built in" and not coming from a plugin, it should be possible to activate it before the plugins are loaded. This would be a refinement for #936, #963.

### PR DESCRIPTION
**Pull request by fatih-erikli**
Fixes #965.

### CODE REVIEW

File: src/fontra/views/editor/editor.js [hunk 1]
```diff
@@ -183,9 +183,29 @@ export class EditorController {
     this.initContextMenuItems();
     this.initShortCuts();
     this.initMiniConsole();
-    this.initPlugins().then(() => {
-      this.restoreOpenTabs("left");
-      this.restoreOpenTabs("right");
+
+    let waitPluginsToRestoreTabs = [];
```
Comment (fatih-erikli) [inline src/fontra/views/editor/editor.js:187]: I'll need a better variable name

Comment (fatih-erikli) [reply src/fontra/views/editor/editor.js:187]: This holds sidebar names

Comment (justvanrossum) [reply src/fontra/views/editor/editor.js:187]: deferredPanels maybe?

Could it be a const?

Comment (justvanrossum) [reply src/fontra/views/editor/editor.js:187]: Yes, it can be a const.

Comment (justvanrossum) [reply src/fontra/views/editor/editor.js:187]: Ah, I know see I misunderstood what waitPluginsToRestoreTabs actually stored. Maybe `deferRestoreOpenTabs`.

**Pull request merged**